lib/rucio/core/rse.py

```diff
@@ -662,39 +658,51 @@ def get_rses_with_attribute_value(key, value, lookup_key, vo='def', session=None
 
 
 @read_session
-def get_rse_attribute(key, rse_id=None, value=None, use_cache=True, session=None):
+def get_rse_attribute_without_cache(rse_id: str, key: str, session: Optional[Session] = None) -> Optional[Union[str, bool]]:
```

Comment:
I would suggest not to expose this function. This is purely for the unit tests and should not be used by anyone else.
Maybe prefix with `__` ?

Reply:
Should be prefixed with a single underscore in that case: https://docs.python.org/3/tutorial/classes.html#private-variables

Reply:
ah, yes.

Reply:
This is a public function. It is called when the `rse_attribute` should be looked up directly, without the cache.

Reply:
Okay, after a talk with @rcarpa I decided to keep the `use_cache` parameter for consistency. I also changed the `get_rse_attribute_without_cache` to a private function.